plugins/uefi-capsule/fu-uefi-backend-freebsd.c

```diff
@@ -92,7 +73,10 @@ fu_uefi_backend_setup (FuBackend *backend, GError **error)
 {
 	g_autofree gchar *efi_ver = fu_kenv_get_string ("efi-version", error);
 	if (efi_ver == NULL) {
-		g_prefix_error (error, "System does not support UEFI mode, no efi-version kenv: ");
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_NOT_SUPPORTED,
+				     "System does not support UEFI mode, no efi-version kenv");
```

Comment:
error is already set by fu_kenv_get_string

Reply:
Right. What's weird, I don't remember touching this function.